doc-files/release-notes-24.md

```diff
@@ -0,0 +1,223 @@
+# Release Notes for JavaFX 24
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+These release notes cover the standalone JavaFX 24 release. JavaFX 24 requires JDK 22 or later. JDK 24 is recommended.
+
+## Important Changes
+
+### JavaFX 24 Requires JDK 22 or Later
+
+JavaFX 24 is compiled with `--release 22` and thus requires JDK 22 or later in order to run. If you attempt to run with an older JDK, the Java launcher will exit with an error message indicating that the `javafx.base` module cannot be read.
+
+See [JDK-8340003](https://bugs.openjdk.org/browse/JDK-8340003) for more information.
+
+### JavaFX Applications Must Use `--enable-native-access`
+
+Running a JavaFX application on JDK 24 will produce a warning from each of the three JavaFX modules that rely on native access, due to the changes specified in [JEP 472](https://openjdk.org/jeps/472). Each warning will include the following message:
+
+```
+WARNING: Restricted methods will be blocked in a future release unless native access is enabled
+```
+
+In order to suppress the warning now, and to be able to run your application at all in a subsequent version of the JDK, you need to explicitly enable native access for all modules that need it. This is done by passing `--enable-native-access=<list-of-modules>` to `java` on the command line, listing the modules that you grant native access. This list of modules includes `javafx.graphics` and, optionally, `javafx.media` and `javafx.web`, if your application uses those modules.
+
+For example:
+
+```
+java --enable-native-access=javafx.graphics,javafx.media,javafx.web
+```
+
+See [JDK-8347744](https://bugs.openjdk.org/browse/JDK-8347744) for more information.
+
+### The `jdk.jsobject` Module is Now Included with JavaFX
+
+The `jdk.jsobject` module, which is used by JavaFX WebView applications, is now included with JavaFX, replacing the JDK module of the same name. The `jdk.jsobject` module is deprecated as of JDK 24, and will be removed in a future release of the JDK.
+
+To facilitate the transition, `jdk.jsobject` is now an upgradable module in the JDK. This means that the version of `jdk.jsobject` delivered with JavaFX can be used in place of the one in the JDK to avoid the compiler warning. This can be done as follows:
+
+#### Applications using the SDK
+
+When running with the JavaFX SDK, use the `--upgrade-module-path` argument. For example:
+
+```
+javac --upgrade-module-path=/path/to/javafx-sdk-24/lib
+java --upgrade-module-path=/path/to/javafx-sdk-24/lib
+```
+
+NOTE: The above will fail if you run your application with JDK 23 or earlier. JDK 24 is recommended when running JavaFX 24, but if you choose to run JavaFX 24 with an earlier JDK, use the `--module-path` option instead.
```

Comment:
Should the `NOTE` be changed to use Github supported [highlight using blockquote](https://github.com/orgs/community/discussions/16925):

```
> [!NOTE]
> The above will fail if you run your application with JDK 23 or earlier. JDK 24 is recommended when running JavaFX 24, but if you choose to run JavaFX 24 with an earlier JDK, use the `--module-path` option instead.
```

Reply:
I'll leave this as is for now, since we are so close to release. I'd also want to check out what GitLab does with it  as well. This might be worth doing for next time.